ui/src/lib/formikhelpers.ts

```diff
@@ -64,6 +64,11 @@ export function parseSubmitValues(values: Experiment) {
           continue
         }
 
+        // Handle container-kill action
+        if (chaos.action === 'container-kill' && action === 'container_name') {
+          continue
+        }
```

Comment:
Though this can fix the problem, I am not sure when we add more chaos types, do we also have to do something similar to this?

Reply:
I have also considered this problem. But since there have some minor questions in API `/experiment/new`, I have to remove other empty actions to prevent them from interfering with the selected.

So, it's easier to remove other actions with a `for each loop`. But to be honest, the configuration file of the chaos experiment is relatively complicated. For now, I think we can allow some **patch** operations, like this PR change.

Admittedly, this approach is not very elegant. After all APIs are stabilized, I think this part of the code can be considered for refactoring.

Reply:
Great. Then let's merge this now.